src/pages/admin/regulatory/forms/CreateRegulatoryForm.js

```diff
@@ -147,7 +147,7 @@ const CreateRegulatoryForm = props => {
 
 
                     {
-                      contacts.map ((contact, index) => 
+                      attachments.map ((attachment, index) =>
```

Comment:
Iterating the attachments object array and generating form elements using field array